pkg/push/api/api.go

```diff
@@ -236,20 +241,149 @@ func (s *PushService) CreatePush(
 		}
 		return nil, dt.Err()
 	}
-	return &pushproto.CreatePushResponse{}, nil
+	return &pushproto.CreatePushResponse{
+		Push: push.Push,
```

Comment:
For security reasons, we must remove the FCM service account from the response

Reference:
https://github.com/bucketeer-io/bucketeer/blob/15a8681012601abfbee96be1a6b849420cbf525d/pkg/push/api/api.go#L942-L945

Reply:
I have remove the FcmServiceAccount from response